lib/src/query_component.dart

```diff
@@ -77,17 +80,17 @@ class QueryComponent {
     try {
       // Uses the queryId to get the report download path.
       final jsonGetQueryResponse =
-      await _queryService.execReportingGetQuery(reportingQueryId);
+          await _queryService.execReportingGetQuery(reportingQueryId);
       final reportingDownloadPath =
-      ReportingQueryParser.parseDownloadPathFromJsonString(
-          json.stringify(jsonGetQueryResponse));
+          ReportingQueryParser.parseDownloadPathFromJsonString(
+              json.stringify(jsonGetQueryResponse));
 
       // Downloads the report and parse the response into a revenue map.
       final report =
-      await _queryService.execReportingDownload(reportingDownloadPath);
+          await _queryService.execReportingDownload(reportingDownloadPath);
 
       return ReportingQueryParser.parseRevenueFromJsonString(report);
-    } catch(e) {
+    } catch (e) {
```

Comment:
has it been throwing? why add this now if not adding handling?

Reply:
The try catch was added in a previous [pr](https://github.com/googleinterns/dv360-excel-plugin/pull/50#discussion_r453093375). I guess I forget to run dartfmt when I merge it, so the indentation was wrong T.T

Reply:
yea but I only just noticed it.
lets merge this for now but think about why this is here. if we are encountering this error then lets figure out why and if we arent we should let it throw so we arent swallowing errors we dont know about.